src/advanced-slots.tsx

```diff
@@ -0,0 +1,350 @@
+
+    value2?: string;
+    text2?: string;
+
+    secondaryValue?: string;
+
+    title?: string;
+    description?: string | Array<string>;
+
+    textColor?: string;
+    'aria-label'?: string;
+}
+
+export const HighlightedValueBlock: React.FC<HighlightedValueBlockProps> = ({
+    tag,
+    value,
+    text,
+    value2,
+    text2,
+    secondaryValue,
+    title,
+    description,
+    textColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div aria-label={ariaLabel}>
+            {tag && (
+                <div
+                    className={sprinkles({
+                        paddingBottom: 24,
+                    })}
+                >
+                    {tag}
+                </div>
+            )}
+
+            <Stack space={2}>
+                <Inline space={8} alignItems="flex-end">
+                    <Text8 color={textColor}>{value}</Text8>
+                    <Text2 regular color={vars.colors.textSecondary}>
+                        {text}
+                    </Text2>
+                </Inline>
+                {value2 && (
+                    <Inline space={8} alignItems="flex-end">
+                        <Text8 color={textColor}>{value2}</Text8>
+                        <Text2 regular color={vars.colors.textSecondary}>
+                            {text2}
+                        </Text2>
+                    </Inline>
+                )}
+                {secondaryValue && <Text8 color={vars.colors.textSecondary}>{secondaryValue}</Text8>}
+            </Stack>
+            <div
+                className={sprinkles({
+                    paddingTop: 8,
+                })}
+            >
+                <SlotContent title={title} description={description} />
+            </div>
+        </div>
+    );
+};
+
+interface ValueBlockProps {
+    title?: string;
+    value?: string;
+    description?: string | Array<string>;
+    textColor?: string;
+    'aria-label'?: string;
+}
+
+export const ValueBlock: React.FC<ValueBlockProps> = ({
+    title,
+    value,
+    description,
+    textColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div
+            aria-label={ariaLabel}
+            style={{
+                display: 'flex',
+                flexDirection: 'column',
+            }}
+        >
+            <Text2 regular color={vars.colors.textPrimary}>
+                {title}
+            </Text2>
+            <Text8 color={textColor}>{value}</Text8>
+            <SlotContent description={description} />
+            {/* <Text2 regular color={vars.colors.textSecondary}>
+                {subtitle}
+            </Text2>
+            <Text2 regular color={vars.colors.textSecondary}>
+                {description}
+            </Text2> */}
+        </div>
+    );
+};
+
+interface ProgressBlockProps {
+    title?: string;
+    stackingGroup?: RendersNullableElement<typeof StackingGroup>;
+
+    progressPercent?: number;
+
```

Comment:
@wyunreal progressBar has a property to reverse animation... should we include a prop here to access to this progress prop? Would it be useful for the CMS to be able to control this for the "te quedan" / "llevas" case?

https://tinyurl.com/2hpfba3y